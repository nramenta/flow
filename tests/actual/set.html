{% set answer_to_life_and_everything = 42 %}

{{ answer_to_life_and_everything }}

{% set partial %}This is a partial{% endset %}

{{ partial }}
