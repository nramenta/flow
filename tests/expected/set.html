

42




This is a partial

